src/transformers/image_transforms.py

```diff
@@ -385,6 +391,11 @@ def normalize(
     if input_data_format is None:
         input_data_format = infer_channel_dimension_format(image)
 
+    # if of ChannelDimension.NONE and not made HWC by resize, handle here
+    if input_data_format == ChannelDimension.NONE:
+        input_data_format = ChannelDimension.LAST
+        image = np.expand_dims(image, axis=-1) if image.ndim == 2 else image
+
     channel_axis = get_channel_dimension_axis(image, input_data_format=input_data_format)
     num_channels = image.shape[channel_axis]
 
```

Comment:
Not sure se should expand dims here, can we handle without expand?

for example:
```python
if input_data_format != ChannelDimension.NONE:
    channel_axis = get_channel_dimension_axis(image, input_data_format=input_data_format)
    num_channels = image.shape[channel_axis]
else:
    num_channels = 1
```